giraffez/commandline.py

```diff
@@ -415,6 +418,9 @@ def run(self, args):
             show_warning(("USING MLOAD TO INSERT LESS THAN {} ROWS IS NOT RECOMMENDED - USE LOAD "
                 "INSTEAD!").format(MLOAD_THRESHOLD), UserWarning)
 
+        if args.delimiter is None:
```

Comment:
MLoad was failing if no delimiter was explicitly given, which is why this was added.  Will look more closely and probably move behind encoding check.